pkg/whitesource/whitesource.go

```diff
@@ -92,22 +92,28 @@ func transformLibToPurlType(libType string) string {
 	log.Entry().Debugf("LibType reported as %v", libType)
 	switch strings.ToLower(libType) {
 	case "java":
+	case "maven_artifact":
```

Comment:
AFAIK the "Java" way does not work here.
You need to separate multiple values with comma.

See e.g. here for various examples: https://yourbasic.org/golang/switch-statement/

maybe also enhance the tests for such a case ...